packages/libspng.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Libspng < Package
+  description 'libspng is a C library for reading and writing Portable Network Graphics (PNG) format files with a focus on security and ease of use.'
+  homepage 'https://libspng.org'
+  version '0.5.0'
+  source_url 'https://gitlab.com/randy408/libspng/uploads/3d980bac86c51368f40af2f1ac79a057/libspng-0.5.0.tar.xz'
+  source_sha256 '220a653802559943ae43fd48f03ba6ff3935a5243766d9ee5ff905240d4399a7'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'cmake' => :build
+  depends_on 'zlibpkg'
```

Comment:
You can exclude these dependencies since they are already in "core".  See https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txthttps://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt.

Reply:
I'll update these tomorrow.